examples/protein_language_modeling.ipynb

```diff
@@ -0,0 +1,2194 @@
+{
```

Comment:
<p>I think an install from source will be needed for now.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/243/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>